Microsoft.Toolkit.Uwp/Helpers/CameraHelper/FrameSourceGroupsHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Media.Capture;
+using Windows.Media.Capture.Frames;
+
+namespace Microsoft.Toolkit.Uwp.Helpers.CameraHelper
+{
+    /// <summary>
+    /// Helper class to get Frame Source Groups that can be used by Media Capture
+    /// </summary>
+    public static class FrameSourceGroupsHelper
```

Comment:
Could this whole helper be part of the camera helper and automaticly do this in the inatialization when the camera helper is initialized, simplifying the API?

Reply:
Modified and moved the code to Camera Helper to simplify initialization.